layers/+lang/c-c++/layers.el

```diff
@@ -10,5 +10,5 @@
 ;;; License: GPLv3
 
 (when (and (boundp 'c-c++-backend)
-           (member c-c++-backend '(lsp-cquery lsp-ccls)))
-  (configuration-layer/declare-layer-dependencies '(lsp dap)))
+           (eq c-c++-backend 'lsp))
+  (configuration-layer/declare-layers '(lsp dap)))
```

Comment:
Use `configuration-layer/declare-layer-dependencies` instead of `configuration-layer/declare-layers`.
If you encounter a bug using it, please let me know and I'll try to fix it.

Reply:
Whoops. Fixed.